src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -334,52 +404,241 @@ private synchronized void writeToConsole(Person person, int index, long time, bo
+
+      // this will have to change based on SPEW latitude/longitude
+      person.attributes.put(Person.CITY, city.city);
+
+      // TODO spew location changes
+      person.attributes.put(Person.STATE, city.state);
+
+      String race = spewPerson.get(randSpew).get("RAC1P");
+      String hisp = spewPerson.get(randSpew).get("HISP");
+
+      // race codes, hispanic is a different variable
+      if (race.equals("1")) {
+        person.attributes.put(Person.RACE, "white");
+      } else if (race.equals("2")) {
+        person.attributes.put(Person.RACE, "black");
+      } else if (race.equals("3")) {
+        person.attributes.put(Person.RACE, "native");
+      } else if (race.equals("4")) {
+        person.attributes.put(Person.RACE, "native");
+      } else if (race.equals("5")) {
+        person.attributes.put(Person.RACE, "native");
+      } else if (race.equals("6")) {
+        person.attributes.put(Person.RACE, "asian");
+      } else if (race.equals("7")) {
+        person.attributes.put(Person.RACE, "asian");
+      } else if (race.equals("8")) {
+        person.attributes.put(Person.RACE, "other");
+      } else if (race.equals("9")) {
+        person.attributes.put(Person.RACE, "other");
+      }
+
+      // TODO hispanic ethnicities that are in SPEW but not in synthea
+
+      if (person.attributes.get(race) == null && !hisp.equals("1")) {
+        person.attributes.put(Person.RACE, "hispanic");
+        person.attributes.put(Person.HISPANIC, true);
+
+        for (int i = 0; i <= hispanicCodes.size() - 1; i++) {
+          if (spewPerson.get(randSpew).get("HISP").equals(hispanicCodes.get(i).get("Code"))) {
+            person.attributes.put(Person.ETHNICITY, hispanicCodes.get(i).get("Ethnicity"));
+          }
+        }
+      }
+
+      if (hisp.equals("1")) {
+        person.attributes.put(Person.HISPANIC, false);
+        String ethnicity = city.ethnicityFromRace((String) person.attributes.get(Person.RACE),
+            person);
+        person.attributes.put(Person.ETHNICITY, ethnicity);
+      }
+
+      String language = city.languageFromEthnicity((String) person.attributes.get(Person.ETHNICITY),
+          person);
+      person.attributes.put(Person.FIRST_LANGUAGE, language);
+
+      String gender = spewPerson.get(randSpew).get("SEX");
+
+      if (gender.equals("1")) {
+        person.attributes.put(Person.GENDER, "M");
+      } else if (gender.equals("2")) {
+        person.attributes.put(Person.GENDER, "F");
+      }
+
+      // Longitude and latitude
+      // TODO a look up to assign address/city/town/zip from lat and long
+      // look into using FIPS codes
+      
+      String latitude = spewPerson.get(randSpew).get("latitude");
+
+      String longitude = spewPerson.get(randSpew).get("longitude");
+       
+      String coordinates = new StringBuilder(latitude).append(",").append(longitude).toString();
+      person.attributes.put(Person.COORDINATE, coordinates);
```

Comment:
the COORDINATE attribute is expected to be a "DirectPosition2D" object - see Person.getLatLon(). This is getting overwritten by LifecycleModule.birth() before anything uses it though so that's why it's not crashing with a ClassCastException